targets/targets.json

```diff
@@ -1712,10 +1712,12 @@
             "SLEEP",
             "SPI",
             "SPISLAVE",
-            "STDIO_MESSAGES"
+            "STDIO_MESSAGES",
+            "USBDEVICE"
```

Comment:
how's enabling USBDEVICE related to Mbed OS 5 ? It should be a separate commit, also includes test logs for this in the PR itself

Reply:
Well, it's related in the sense that USB Device was already supported in Mbed OS 2, and for some reason it worked even though USBDEVICE was not declared in the .json file. If you notice, I declared USBDEVICE but I didn't have to add any .c code to support it, it's because it's all already there. However if you still think I should separate it out, I'm fine with it. Should I force-push to this branch?

Regarding test logs -- Is there some specific testing that I should do that I should read up on? I tried both USBSerial, and USBMSD with a dummy buffer, and they worked fine.

Reply:
I've checked ,correct was already implemented. All fine then